server/api/auth/getInvitation.ts

```diff
@@ -0,0 +1,20 @@
+import { Express, Response, Request } from 'express'
+import { fetchInvitation, findUserByEmail } from '@server/repository/user/userRepository'
+import { sendErr } from '@server/utils/requestUtils'
+import { ApiEndPoint } from '@server/api/endpoint'
```

Comment:
apiEndpoint can be used by client as well. should it go under common?

Reply:
Thought so too, i'll move it to common